robottelo/ui/session.py

```diff
@@ -28,12 +28,13 @@ def __enter__(self):
         return self
 
     def __exit__(self, exc_type, exc_value, traceback):
-        self.logout()
+        #self.logout()
+        pass
```

Comment:
Let's not logout, otherwise we are unable to capture screenshot.
I think `tearDown()` function in `robottelo/test.py would anyways quit the browser for us.